package.yaml

```diff
@@ -1,7 +1,7 @@
 name: "tasty-discover"
-version: "2.0.1"
+version: "2.0.2"
 synopsis: "Test discovery for the tasty framework."
-description: "Test discovery for the tasty framework."
+description: "Please see the README.md below for usage information."
```

Comment:
This prompted me to search around again to understand what the hell these fields are actually for. I [liked this email](https://mail.haskell.org/pipermail/haskell-cafe/2016-November/125541.html). So, it seems I should give something longer than the synopsis and less in-depth than the README :smile: